Jenkinsfile

```diff
@@ -122,14 +122,23 @@ dockerizedBuildPipeline(
     )
   },
   deploy: {
+    def version = env.VERSION
+    def majorVersion = version.substring(0, version.indexOf('.'))
+    def tag = isMainBranch() ? null : "latest-$majorVersion"
+    def tagArg = tag ? "--tag $tag" : ''
+
+    def doPublish = {
+      sh "npm publish --access public $tagArg lib/dist/sonatype-react-shared-components-\$VERSION.tgz"
+    }
+
+    // publish to repo.s.c. We must do this in addition to publishing to npmjs.com because of the
+    // namespace confusion protection feature of Nexus Firewall. See RSC-1430 and the slack conversation
+    // linked therein
+    doPublish()
+
+    // publish to npmjs.com
     withCredentials([string(credentialsId: 'uxui-npm-auth-token', variable: 'NPM_TOKEN')]) {
-      withDockerImage(env.DOCKER_IMAGE_ID, 'npmjs-npmrc') {
-        def version = env.VERSION
-        def majorVersion = version.substring(0, version.indexOf('.'))
-        def tag = isMainBranch() ? null : "latest-$majorVersion"
-        def tagArg = tag ? "--tag $tag" : ''
-        sh "npm publish --access public $tagArg lib/dist/sonatype-react-shared-components-\$VERSION.tgz"
-      }
+      withDockerImage(env.DOCKER_IMAGE_ID, 'npmjs-npmrc', doPublish)
```

Comment:
Use the `rsc-internal-write-npmrc` npmrc config id for publishing.

Reply:
> Use the rsc-internal-write-npmrc npmrc config id for publishing.

[062c905](https://github.com/sonatype/sonatype-react-shared-components/pull/944/commits/062c9056384c91e820b783e0a5da1e2ddef9284c)